public/app/features/explore/Explore.tsx

```diff
@@ -190,37 +191,33 @@ export class Explore extends React.PureComponent<Props, ExploreState> {
    * TODO: In the future, we would like to return active filters based the query that produced the log line.
    * @alpha
    */
-  isFilterLabelActive = async (key: string, value: string) => {
+  isFilterLabelActive = async (key: string, value: string, row: LogRowModel) => {
```

Comment:
Could we pass `refId` instead of `LogRowModel` instead to make it easier to re-use by other visualizations in the future?